### PR TITLE
Add README to fuzzing set up

### PR DESCRIPTION
**Pull request by DavidKorczynski**
Add README for the fuzzers following the suggestion of @bvanassche here: https://github.com/net-snmp/net-snmp/pull/321#issuecomment-889286308

### CODE REVIEW

File: fuzzing/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Fuzzing
+This folder contains a fuzzing setup that is meant to be used with OSS-Fuzz. 
+
+Fuzzing as a concept aims to stress-test the code under analysis. Traditional fuzzing is based on sending random input to the target application, whereas modern fuzzing relies on genetic algorithms based on code-coverage. The goal of the fuzzing in net-snmp is to catch any potential bugs, in particular memory corruption issues. The net-snmp code is instrumented with different sanitizers, e.g. AddressSanitizer, when being fuzzed.
```
Comment (bvanassche) [inline fuzzing/README.md:4]: All other Net-SNMP source code and README documents only use 80 columns. Should the width of this text be restricted to 80 columns too (except for hyperlinks)?

**Comment by bvanassche** [comment]
Please use the imperative mood in the subject (Adds -> Add) and remove the trailing dot from the subject of this patch. See also https://chris.beams.io/posts/git-commit/

File: fuzzing/README.md (hunk 1 above)
Comment (DavidKorczynski) [reply fuzzing/README.md]: I think they should - fixed it up now.

**Comment by bvanassche** [comment]
Thanks!

**Pull request merged**